students/stuartkershaw/lesson02/fizzbuzz.py

```diff
@@ -0,0 +1,13 @@
+counter = 1
+limit = 100
+
+while counter <= limit:
```

Comment:
take a look at `range` function in python, it makes these like this a little bit easier (not having to keep track of counter)